backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlRepository.java

```diff
@@ -156,14 +153,14 @@ public final boolean insertTranslations(@NonNull final POTrlInfo trlInfo, final
 		}
```

Comment:
Please check out the two differnt where-clauses: 
as talked yesterday, this whole if doesn't make any sense anymore.

Reply:
There might be languages that are marked as isbaselanguage = 'y' and issystemlanguage='n' that we need trls for for the ad_element table. That's the scenario I left the if for. WDYT?

Reply:
aaah, ok, i think now i got it.
then i think we should change this
```java
		if (I_AD_Element.Table_Name.equals(tableName))
		{
			sql.append("AND (l." + I_AD_Language.COLUMNNAME_IsSystemLanguage + "='Y' OR l." + I_AD_Language.COLUMNNAME_IsBaseLanguage + "='Y')");
		}
		else
		{
			sql.append("AND (l." + I_AD_Language.COLUMNNAME_IsSystemLanguage + "='Y')");
		}
```

to
```java
		sql.append("AND (l." + I_AD_Language.COLUMNNAME_IsSystemLanguage + "='Y'");
		if (I_AD_Element.Table_Name.equals(tableName))
		{ // for AD_Element we need to make sure to have the BaseLanguage, no matter if that's also a system-language
			sql.append(" OR l." + I_AD_Language.COLUMNNAME_IsBaseLanguage + "='Y'");
		}
		sql.append(")");
```

i think that way it's clearer